project_disable_last_sol_as_sol_by_default/__manifest__.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2022 Stéphan Sainléger (Elabore)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "project_disable_last_sol_as_sol_by_default",
+    "version": "14.0.1.0.0",
+    "author": "Elabore",
+    "website": "https://elabore.coop",
+    "maintainer": "Laetitia Da Costa",
+    "license": "AGPL-3",
+    "category": "Project",
+    "summary": "Do not add last sale order line automaticly as the sale order line by default in a task",
+    "description": """
```

Comment:
@LaetitiaElabore The description field should be deleted (especially when it deals with another add-on :) )